src/Uno.UI/UI/Xaml/Controls/MenuFlyout/MenuFlyout.cs

```diff
@@ -172,16 +172,6 @@ private protected override void OnOpening()
 			}
 		}
 
-		internal override void OnClosing(ref bool cancel)
-		{
-			base.OnClosing(ref cancel);
-
-			if (!cancel)
-			{
-				CloseSubMenu();
```

Comment:
Is this call not needed? I don't see it being moved somewhere else

Reply:
It's already happening in Closed.